### PR TITLE
Adjust pointers to sections

### PR DESCRIPTION
**Pull request by mirjak**

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -946,13 +946,14 @@ latest_rtt:
 
 smoothed_rtt:
 : The smoothed RTT of the connection, computed as described in
-  {{?RFC6298}}
+  Section {{smoothed-rtt}}
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]:
```suggestion
  Section {{smoothed-rtt}}.
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -946,13 +946,14 @@ latest_rtt:
...
 
 rttvar:
-: The RTT variation, computed as described in {{?RFC6298}}
+: The RTT variation, computed as described in Section {{smoothed-rtt}}
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]:
```suggestion
: The RTT variation, computed as described in Section {{smoothed-rtt}}.
```

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -946,13 +946,14 @@ latest_rtt:
...
 
 min_rtt:
-: The minimum RTT seen in the connection, ignoring ack delay.
+: The minimum RTT seen in the connection, ignoring ack delay, as described
+  in Section {{min-rtt}}
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]:
```suggestion
  in Section {{min-rtt}}.
```

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-recovery.md]: I think that you mean
```suggestion
  {{smoothed-rtt}}.
```

(Section references include the string "Section ").

File: draft-ietf-quic-recovery.md (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-quic-recovery.md]:
```suggestion
: The RTT variation, computed as described in {{smoothed-rtt}}
```

File: draft-ietf-quic-recovery.md (hunk 3 above)
Comment (martinthomson) [reply draft-ietf-quic-recovery.md]:
```suggestion
  in {{min-rtt}}.
```

**Pull request merged**